yarn-project/aztec/src/cli/cmds/start_node.ts

```diff
@@ -50,7 +50,7 @@ export async function startNode(
 
   const testAccounts = nodeConfig.testAccounts ? (await getInitialTestAccounts()).map(a => a.address) : [];
   const sponsoredFPCAccounts = nodeConfig.sponsoredFPC ? [await getSponsoredFPCAddress()] : [];
-  const initialFundedAccounts = [...testAccounts, ...sponsoredFPCAccounts];
+  const initialFundedAccounts = testAccounts.concat(sponsoredFPCAccounts);
```

Comment:
I was paranoid that I hadn't used the same concat method used when creating the genesis state in the cli deployment of L1 contracts.